kube/src/config/mod.rs

```diff
@@ -135,7 +138,7 @@ impl Config {
             }
         }
 
-        match loader.identity(" ") {
+        match loader.identity(IDENTITY_PASSWORD) {
```

Comment:
is this equivalent? it was single space before, now it's empty string.

Reply:
`const IDENTITY_PASSWORD: &'static str = " ";`
It's still a single space

Reply:
oh, doh. I saw a blank string default init elsewhere.